packages/components/src/client/styles/repl-it.scss

```diff
@@ -0,0 +1,35 @@
+@use "vuepress-shared/styles/reset";
+
+.replit-wrapper {
+  margin: 1rem auto;
+  border: none;
+  text-align: center;
+  transition: background var(--t-color, 0.3s ease);
+
+  .replit-button {
+    @include reset.button;
+
+    display: inline-block;
+
+    margin: 0 auto;
+    padding: 0.5rem 1rem;
+    border-radius: 0.5rem;
+
+    background: var(--c-bg-light);
+    color: var(--c-text);
```

Comment:
Fallback color should be provided in case users are not using default theme

Reply:
Fixed.